special-interest-groups/sig-exec/roles-and-organization-management.md

```diff
@@ -1,34 +1,34 @@
-# SIG Expr Roles and Organization Management
+# SIG Exec Roles and Organization Management
 
 ## Join and Promotion
 
 ### Join
 
-The minimal requirement for contributors to join the Expr SIG is to have
-the **Active Contributor** community role in the expression package.
+The minimal requirement for contributors to join the Exec SIG is to have
+the **Active Contributor** community role in the executor/expression package.
 
 ### Promotion
 
 #### From `Contributor` to `Active Contributor`
 
 Requirements:
 
-* Have at least 8 PRs merged in the expression package within one year.
-* Nominated by at least 2 Expr SIG Reviewers, Committer or Maintainer.
+* Have at least 8 PRs merged in the expression package or 4 PRs merged in the executor package within one year.
```

Comment:
@qw4990 should we obey the common definition about `active contributor`?
@dcalvin please take a look.

Reply:
Yeah here we only list the SIG specific requirements. If there is nothing specific here, we should just refer to the common definition.